libs/designsystem/src/lib/components/checkbox/checkbox.component.spec.ts

```diff
@@ -47,20 +59,50 @@ describe('CheckboxComponent', () => {
       expect(spectator.element).toHaveClass('attention-level1');
     });
 
+    it('should have correct icon styling when attentionLevel is 1', () => {
+      spectator.setInput('attentionLevel', '1');
+      spectator.detectChanges();
+      expect(ionCheckbox).toHaveComputedStyle({
+        '--checkmark-color': getColor('black'),
+        '--background-checked': getColor('success'),
+        '--border-color-checked': getColor('success'),
+      });
+    });
+
```

Comment:
Consider either merging this with `it('should have class .attention-level1 when attentionLevel is 1)'´ above (as it's basically the same tihng you're testing) or removing the css-class test altogether (as it's only tests an interim thing...)